CI: Keeping an Eye on macOS + Fortran

Opening this issue so we can post log files from macOS CI runners that failed while building with Fortran. Migrated from #1004

Please post links to PRs and attach text files with logs when you spot them. In the "Checks" tabs, e.g. [here](https://github.com/AMReX-Codes/amrex/pull/1004/checks?check_run_id=763167004), is a `...` symbol that has the option "View raw logs".

https://github.com/AMReX-Codes/amrex/pull/1111/checks?check_run_id=848249709

Linux + Fortran also fails.

```
...
2020-07-08T03:30:38.0840557Z [ 74%] Building Fortran object Tutorials/Amr/Advection_AmrLevel/CMakeFiles/Advection_AmrLevel_SV.exe.dir/Source/Src_nd/tagging_params.f90.o
2020-07-08T03:30:38.0841183Z [ 74%] Building Fortran object Tutorials/Amr/Advection_AmrLevel/CMakeFiles/Advection_AmrLevel_UV.exe.dir/Source/Src_nd/tagging_params.f90.o
2020-07-08T03:30:38.0844548Z cd /home/runner/work/amrex/amrex/build/Tutorials/Amr/Advection_AmrLevel && /usr/bin/gfortran -DAMREX_DEBUG -DAMREX_GIT_VERSION=\"20.07.0\" -DAMREX_LANG_FORT -DAMREX_Linux -DAMREX_PARTICLES -DAMREX_SPACEDIM=3 -DAMREX_USE_ASSERTION -DAMREX_USE_MPI -DBL_FORT_USE_UNDERSCORE -DBL_LANG_FORT -DBL_Linux -DBL_SPACEDIM=3 -DBL_USE_MPI -I/home/runner/work/amrex/amrex/Tutorials/Amr/Advection_AmrLevel/Source -I/home/runner/work/amrex/amrex/build/Tutorials/Amr/Advection_AmrLevel/mod_files -I/home/runner/work/amrex/amrex/build/mod_files -I/home/runner/work/amrex/amrex/Src/Base -I/home/runner/work/amrex/amrex/Src/Boundary -I/home/runner/work/amrex/amrex/Src/AmrCore -I/home/runner/work/amrex/amrex/Src/Amr -I/home/runner/work/amrex/amrex/Src/LinearSolvers/MLMG -I/home/runner/work/amrex/amrex/Src/LinearSolvers/Projections -I/home/runner/work/amrex/amrex/Src/Particle -I/usr/lib/x86_64-linux-gnu/openmpi/include/openmpi -I/usr/lib/x86_64-linux-gnu/openmpi/include/openmpi/opal/mca/event/libevent2022/libevent -I/usr/lib/x86_64-linux-gnu/openmpi/include/openmpi/opal/mca/event/libevent2022/libevent/include -I/usr/lib/x86_64-linux-gnu/openmpi/include  -g -Jmod_files   -O0 -ggdb -fcheck=bounds -fbacktrace -Wuninitialized -Wunused -finit-real=snan -finit-integer=2147483647 -fimplicit-none -pthread -ffixed-line-length-none -ffree-line-length-none -c /home/runner/work/amrex/amrex/Tutorials/Amr/Advection_AmrLevel/Source/Src_nd/tagging_params.f90 -o CMakeFiles/Advection_AmrLevel_UV.exe.dir/Source/Src_nd/tagging_params.f90.o
2020-07-08T03:30:38.0849685Z cd /home/runner/work/amrex/amrex/build/Tutorials/Amr/Advection_AmrLevel && /usr/bin/gfortran -DAMREX_DEBUG -DAMREX_GIT_VERSION=\"20.07.0\" -DAMREX_LANG_FORT -DAMREX_Linux -DAMREX_PARTICLES -DAMREX_SPACEDIM=3 -DAMREX_USE_ASSERTION -DAMREX_USE_MPI -DBL_FORT_USE_UNDERSCORE -DBL_LANG_FORT -DBL_Linux -DBL_SPACEDIM=3 -DBL_USE_MPI -I/home/runner/work/amrex/amrex/Tutorials/Amr/Advection_AmrLevel/Source -I/home/runner/work/amrex/amrex/build/Tutorials/Amr/Advection_AmrLevel/mod_files -I/home/runner/work/amrex/amrex/build/mod_files -I/home/runner/work/amrex/amrex/Src/Base -I/home/runner/work/amrex/amrex/Src/Boundary -I/home/runner/work/amrex/amrex/Src/AmrCore -I/home/runner/work/amrex/amrex/Src/Amr -I/home/runner/work/amrex/amrex/Src/LinearSolvers/MLMG -I/home/runner/work/amrex/amrex/Src/LinearSolvers/Projections -I/home/runner/work/amrex/amrex/Src/Particle -I/usr/lib/x86_64-linux-gnu/openmpi/include/openmpi -I/usr/lib/x86_64-linux-gnu/openmpi/include/openmpi/opal/mca/event/libevent2022/libevent -I/usr/lib/x86_64-linux-gnu/openmpi/include/openmpi/opal/mca/event/libevent2022/libevent/include -I/usr/lib/x86_64-linux-gnu/openmpi/include  -g -Jmod_files   -O0 -ggdb -fcheck=bounds -fbacktrace -Wuninitialized -Wunused -finit-real=snan -finit-integer=2147483647 -fimplicit-none -pthread -ffixed-line-length-none -ffree-line-length-none -c /home/runner/work/amrex/amrex/Tutorials/Amr/Advection_AmrLevel/Source/Src_nd/tagging_params.f90 -o CMakeFiles/Advection_AmrLevel_SV.exe.dir/Source/Src_nd/tagging_params.f90.o
2020-07-08T03:30:38.0918143Z f951: Fatal Error: Can't rename module file ‘mod_files/tagging_params_module.mod0’ to ‘mod_files/tagging_params_module.mod’: No such file or directory
...
2020-07-08T03:30:42.4624308Z /usr/local/bin/cmake -E cmake_copy_f90_mod Tutorials/Amr/Advection_AmrLevel/mod_files/tagging_params_module.mod Tutorials/Amr/Advection_AmrLevel/CMakeFiles/Advection_AmrLevel_UV.exe.dir/tagging_params_module.mod.stamp GNU
2020-07-08T03:30:42.4635419Z Error copying Fortran module "Tutorials/Amr/Advection_AmrLevel/mod_files/tagging_params_module.mod".  Tried "Tutorials/Amr/Advection_AmrLevel/mod_files/TAGGING_PARAMS_MODULE.mod" and "Tutorials/Amr/Advection_AmrLevel/mod_files/tagging_params_module.mod".
2020-07-08T03:30:42.4637185Z make[3]: *** [Tutorials/Amr/Advection_AmrLevel/CMakeFiles/Advection_AmrLevel_UV.exe.dir/tagging_params_module.mod.stamp] Error 1
2020-07-08T03:30:42.4637820Z Tutorials/Amr/Advection_AmrLevel/CMakeFiles/Advection_AmrLevel_UV.exe.dir/depend.make:584: recipe for target 'Tutorials/Amr/Advection_AmrLevel/CMakeFiles/Advection_AmrLevel_UV.exe.dir/tagging_params_module.mod.stamp' failed
```

[linux_gfortran.log](https://github.com/AMReX-Codes/amrex/files/4888474/linux_gfortran.log)

--- a/Tutorials/Amr/Advection_AmrLevel/CMakeLists.txt
+++ b/Tutorials/Amr/Advection_AmrLevel/CMakeLists.txt
@@ -39,9 +39,9 @@ target_sources ( ${EXENAME}
 
 set_target_properties ( ${EXENAME} PROPERTIES
    INCLUDE_DIRECTORIES
-   "${INC1};${INC2};${INC3};${CMAKE_CURRENT_BINARY_DIR}/mod_files"
+   "${INC1};${INC2};${INC3};${CMAKE_CURRENT_BINARY_DIR}/${EXENAME}_mod_files"
    Fortran_MODULE_DIRECTORY
-   ${CMAKE_CURRENT_BINARY_DIR}/mod_files
+   ${CMAKE_CURRENT_BINARY_DIR}/${EXENAME}_mod_files
    RUNTIME_OUTPUT_DIRECTORY
    ${CMAKE_CURRENT_BINARY_DIR}/SingleVortex )
 
@@ -85,9 +85,9 @@ target_sources ( ${EXENAME}
 
 set_target_properties ( ${EXENAME} PROPERTIES
    INCLUDE_DIRECTORIES
-   "${INC1};${INC2};${INC3};${CMAKE_CURRENT_BINARY_DIR}/mod_files"
+   "${INC1};${INC2};${INC3};${CMAKE_CURRENT_BINARY_DIR}/${EXENAME}_mod_files"
    Fortran_MODULE_DIRECTORY
-   ${CMAKE_CURRENT_BINARY_DIR}/mod_files
+   ${CMAKE_CURRENT_BINARY_DIR}/${EXENAME}_mod_files
    RUNTIME_OUTPUT_DIRECTORY
    ${CMAKE_CURRENT_BINARY_DIR}/UniformVelocity )
 

--- a/Tutorials/SENSEI/Advection_AmrLevel/CMakeLists.txt
+++ b/Tutorials/SENSEI/Advection_AmrLevel/CMakeLists.txt
@@ -37,9 +37,9 @@ target_sources ( ${EXENAME}
 
 set_target_properties ( ${EXENAME} PROPERTIES      
    INCLUDE_DIRECTORIES
-   "${INC1};${INC2};${INC3};${CMAKE_CURRENT_BINARY_DIR}/mod_files"
+   "${INC1};${INC2};${INC3};${CMAKE_CURRENT_BINARY_DIR}/${EXENAME}_mod_files"
    Fortran_MODULE_DIRECTORY
-   ${CMAKE_CURRENT_BINARY_DIR}/mod_files
+   ${CMAKE_CURRENT_BINARY_DIR}/${EXENAME}_mod_files
    RUNTIME_OUTPUT_DIRECTORY
    ${CMAKE_CURRENT_BINARY_DIR}/SingleVortex )
 
@@ -80,9 +80,9 @@ target_sources ( ${EXENAME}
 
 set_target_properties ( ${EXENAME} PROPERTIES      
    INCLUDE_DIRECTORIES
-   "${INC1};${INC2};${INC3};${CMAKE_CURRENT_BINARY_DIR}/mod_files"
+   "${INC1};${INC2};${INC3};${CMAKE_CURRENT_BINARY_DIR}/${EXENAME}_mod_files"
    Fortran_MODULE_DIRECTORY
-   ${CMAKE_CURRENT_BINARY_DIR}/mod_files
+   ${CMAKE_CURRENT_BINARY_DIR}/${EXENAME}_mod_files
    RUNTIME_OUTPUT_DIRECTORY
    ${CMAKE_CURRENT_BINARY_DIR}/UniformVelocity )
 

